packages/core/src/btc/wallets.ts

```diff
@@ -153,4 +154,31 @@ export class BtcWallets extends Wallets<BtcMasterWallet> {
       this.env
     );
   }
+
+  async createInactiveMasterWallet(
+    name: string
+  ): Promise<InactiveMasterWallet> {
+    checkNullAndUndefinedParameter({ name });
+    const encryptionKeyBuffer: Buffer = this.createEncryptionKey("dummy");
```

Comment:
how about make readonly DUMMY variable to use "dummy" ?

Reply:
I've changed this to `createDummyEncryptionKey()` method. It makes `ffffff...` having length 64.